docs/source/common/trainer.rst

```diff
@@ -1746,6 +1746,20 @@ The current logger being used. Here's an example using tensorboard
         tensorboard = logger.experiment
 
 
+loggers (p)
+***********
+
+The list of loggers currently being used.
+
+.. code-block:: python
+
+    def training_step(self, batch, batch_idx):
+        # List of LightningLoggerBase objects
+        loggers = trainer.loggers
+        for logger in loggers:
+            logger.log_metrics({"foo": 1.0})
+
```

Comment:
`training_step` implies this is being implemented within the `LightningModule`. 
The LightningModule now has a `loggers` property as well
So it doesn't make sense to refer to self.trainer.loggers inside of the Lightning Module if users can already do self.loggers

that's why I think the training_step framing doesn't make sense

Reply:
Oh I understand now. I found it strange that the documentation showed self.trainer.logger in the context of training_step as well. That one needs to be fixed too.